source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -948,6 +1146,33 @@ export default class WalletSendForm extends Component<Props, State> {
                   </div>
                 )}
               </div>
+              {this.state.adaInputState === AdaInputStateType.Updated && (
+                <div
+                  className={styles.minimumAmountNotice}
+                  data-testid="WalletSendForm::minimumAmountNotice::updated"
+                >
+                  <FormattedHTMLMessage
+                    {...messages.minimumAmountNotice}
+                    values={{
+                      minimumAda: minimumAdaValue,
+                    }}
+                  />
+                </div>
+              )}
+              {this.state.adaInputState === AdaInputStateType.Restored && (
+                <div
+                  className={styles.minimumAmountNotice}
+                  data-testid="WalletSendForm::minimumAmountNotice::restored"
+                >
+                  <FormattedHTMLMessage
+                    {...messages.restoredAdaAmount}
+                    values={{
+                      minimumAda: minimumAdaValue,
+                      adaAmount: adaAmountField.value,
+                    }}
+                  />
+                </div>
+              )}
```

Comment:
You can merge those two cases in one, however I am not sure about its readability (it is hard to see it clearly on the GH), so just a suggestion :)

```suggestion
              {[AdaInputStateType.Updated, AdaInputStateType.Restored].includes(this.state.adaInputState) && (
                <div
                  className={styles.minimumAmountNotice}
                  data-testid={`WalletSendForm::minimumAmountNotice::${this.state.adaInputState === AdaInputStateType.Updated ? 'updated' : 'restored}`}
                >
                  <FormattedHTMLMessage
                    {...(this.state.adaInputState === AdaInputStateType.Updated && messages.minimumAmountNotice)}
                    {...(this.state.adaInputState === AdaInputStateType.Restored && messages.minimumAmountNotice)}
                    values={{
                      minimumAda: minimumAdaValue,
                      ...(this.state.adaInputState === AdaInputStateType.Restored && {adaAmount: adaAmountField.value})
                    }}
                  />
                </div>
              )}
```